src/node_crypto.cc

```diff
@@ -741,6 +743,63 @@ void SecureContext::AddCRL(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void UseExtraCaCerts(const char* file) {
+  free(extra_root_certs_file);
+  extra_root_certs_file = file ? strdup(file) : nullptr;
```

Comment:
Since you're making a copy, you might as well change `extra_root_certs_file` to `std::string`.